src/dashboard/src/pages/ChainCode/ChainCode.js

```diff
@@ -335,14 +335,14 @@ class ChainCode extends PureComponent {
             <Divider type="vertical" />
             <a>
               {intl.formatMessage({
-                id: 'app.operator.chainCode.table.operate.approve',
+                id: 'app.chainCode.table.operate.approve',
                 defaultMessage: 'Approve',
               })}
             </a>
             <Divider type="vertical" />
             <a>
               {intl.formatMessage({
-                id: 'app.operator.chainCode.table.operate.commit',
+                id: 'app.chainCode.table.operate.commit',
```

Comment:
Can't find "app.chainCode.table.operate.commit" in locales

Reply:
Also in #572

Reply:
Ok